docs/extensions/getting-started.md

```diff
@@ -13,9 +13,9 @@ Use the [custom blocks feature](/defining-blocks) for this purpose.
 
 You will need to get a [GitHub](https://github.com) account and create a GitHub repository. At this time, other repository hosting solutions (GitLab, Bitbucket, etc.) are not supported.
 
-From the main editor screen, go to the gear wheel and **Extensions**. Click on **GitHub login**
-at the bottom of the list
-and follow prompts to generate a personal access token (make sure to select the **repo** checkbox). 
+Create a new project from the home screen. In the editor, click on the **GitHub** button next to the
+**Save Project** window and button. You'll then be prompted to sign in to GitHub (if you don't already
+have a GitHub account, you can create one by clicking the link on the sign in page).
 Once logged in, the Import dialog 
```

Comment:
@abchatra  - That's kind of for the Import context. What's the guidance for sign in for creating an extension with a commit/push to GitHub?

Reply:
I think you are right @ganicke. What I suggested is import context. Github icon is better to create a new extension. I approved this PR>